README.md

```diff
@@ -20,10 +20,10 @@ We're building a community for discovery and learning.
 
 ## How do I contribute?
 Great question! You are expected to fork this repository and to create PRs when you want to merge your changes. Your commits are expected to follow our [Commit Convention](https://github.com/ThinkFiveable/open/blob/main/CONTRIBUTING.md#commiting), so some example ones are:
-- `fix: broken button on index.html`  
-- `fix(index): broken button`  
-- `chore: add NEWFILE.md`  
-- `feat: new button!!!`   
+- `fix: broken button on index.html`
+- `fix(index): broken button`
```

Comment:
```suggestion
- `fix(Index): broken button`
```
can we use this PR to fix the commit message example issue that you brought up to me?

Reply:
sure